modules/@apostrophecms/ui/ui/apos/components/AposSpinner.vue

```diff
@@ -110,18 +110,22 @@ module.exports = {
 
 @keyframes clockwise {
   from {
+    // stylelint-disable-next-line property-no-vendor-prefix
```

Comment:
@gregvanbrug did you have a specific reason for keeping the `webkit` prefix?

Reply:
Nope. Should be dropped.

Reply:
👍